modules/client/src/client-common/modules/web3.ts

```diff
@@ -207,23 +126,9 @@ export class Web3Module implements IClientWeb3Core {
 
   /** Returns the current DAO factory address */
   public getDaoFactoryAddress(): string {
-    if (!this.daoFactoryAddress) {
+    if (!this.context.daoFactoryAddress) {
       throw new NoDaoFactory();
     }
-    return this.daoFactoryAddress;
-  }
-
-  public getPluginRepoRegistryAddress(): string {
-    if (!this.pluginRepoRegistryAddress) {
-      throw new NoPluginRepoRegistry();
-    }
-    return this.pluginRepoRegistryAddress;
-  }
-
-  public getDaoRegistry(): string {
-    if (!this.daoRegistryAddress) {
-      throw new NoDaoRegistry();
-    }
-    return this.daoRegistryAddress;
```

Comment:
Why removing these?

Reply:
ping

Reply:
We dont use them in the code so it made no sense to leave them there, besides we are going to have multiple pluginRepos depending on the plugin type